### ISSUE TITLE
Fix tests.foreman.ui.test_config_groups.ConfigGroups.test_delete_positive_1

### ISSUE THREAD

**Issue by kbidarkar** [SatelliteQE/robottelo#2036]
```
File "/usr/lib64/python2.7/unittest/case.py", line 369, in run
    testMethod()
  File "/home/jenkins/shiningpanda/jobs/abb3bf3f/virtualenvs/d41d8cd9/lib/python2.7/site-packages/ddt.py", line 146, in wrapper
    return func(self, *args, **kwargs)
  File "/home/jenkins/workspace/satellite6-automation-downstream-rhel66-ui/tests/foreman/ui/test_config_groups.py", line 130, in test_delete_positive_1
    (common_locators["notif.success"]))
  File "/usr/lib64/python2.7/unittest/case.py", line 1000, in assertIsNotNone
    self.fail(self._formatMessage(msg, standardMsg))
  File "/usr/lib64/python2.7/unittest/case.py", line 450, in fail
    raise self.failureException(msg)
'unexpectedly None\n-------------------- >> begin captured logging << --------------------\nrobottelo.ui.locators: DEBUG: Accessing locator "login.username" by id: "login_login"\nrobottelo.ui.locators: DEBUG: Accessing locator "login.username" by id: "login_login"\nrobottelo.ui.locators: DEBUG: Accessing locator "login.password" by id: "login_password"\nrobottelo.ui.locators: DEBUG: Accessing locator "submit" by name: "commit"\nrobottelo.ui.locators: DEBUG: Accessing locator "notif.error" by xpath: "//div[contains(@class, \'jnotify-notification-error\')]"\nrobottelo.ui.locators: DEBUG: Accessing locator "menu.configure" by xpath: "//div[contains(@style,\'static\') or contains(@style, \'fixed\')]//a[@id=\'configure_menu\']"\nrobottelo.ui.locators: DEBUG: Accessing locator "menu.configure_groups" by xpath: "//div[contains(@style,\'static\') or contains(@style, \'fixed\')]//a[@id=\'menu_item_config_groups\']"\nrobottelo.ui.locators: DEBUG: Accessing locator "config_groups.new" by xpath: "//a[@data-id=\'aid_config_groups_new\']"\nrobottelo.ui.locators: DEBUG: Accessing locator "config_groups.name" by xpath: "//input[@id=\'config_group_name\']"\nrobottelo.ui.locators: DEBUG: Accessing locator "config_groups.name" by xpath: "//input[@id=\'config_group_name\']"\nrobottelo.ui.locators: DEBUG: Accessing locator "submit" by name: "commit"\nrobottelo.ui.locators: DEBUG: Accessing locator "menu.configure" by xpath: "//div[contains(@style,\'static\') or contains(@style, \'fixed\')]//a[@id=\'configure_menu\']"\nrobottelo.ui.locators: DEBUG: Accessing locator "menu.configure_groups" by xpath: "//div[contains(@style,\'static\') or contains(@style, \'fixed\')]//a[@id=\'menu_item_config_groups\']"\nrobottelo.ui.locators: DEBUG: Accessing locator "config_groups.select_name" by xpath: "//a[contains(.,\'%s\') and contains(@href, \'edit\')]"\nrobottelo.ui.locators: DEBUG: Accessing locator "search" by id: "search"\nrobottelo.ui.locators: DEBUG: Accessing locator "search_button" by xpath: "//button[contains(@type,\'submit\')]"\nrobottelo.ui.locators: DEBUG: Accessing locator "config_groups.dropdown" by xpath: "//td/a[normalize-space(.)=\'%s\']/following::td/div/a[@data-toggle=\'dropdown\']"\nrobottelo.ui.locators: DEBUG: Accessing locator "config_groups.select_name" by xpath: "//a[contains(.,\'%s\') and contains(@href, \'edit\')]"\nrobottelo.ui.locators: DEBUG: Accessing locator "config_groups.delete" by xpath: "//a[@class=\'delete\' and contains(@data-confirm, \'%s\')]"\nrobottelo.ui.locators: DEBUG: Accessing locator "search" by id: "search"\nrobottelo.ui.locators: DEBUG: Accessing locator "search_button" by xpath: "//button[contains(@type,\'submit\')]"\nrobottelo.ui.locators: DEBUG: Accessing locator "notif.success" by xpath: "//div[contains(@class, \'jnotify-notification-success\')]"\n--------------------- >> end captured logging << ---------------------'
```

**Comment by sghai** [comment]
Fixed in PR #2056

**Comment by elyezer** [comment]
#2056 got merged, closing this.
